### PR TITLE
The system should return already-aggregated results from Elasticsearch

### PR DESCRIPTION
**Pull request by FerPerales**
Asana ticket: https://app.asana.com/0/1203289004376659/1205005445681275/f

This PR:
* Update OccupationStandardElasticsearchQuery class to retrieve collapse results
* Add InnerHit PORO model to wrap inner_hits values from query mentioned above 
* Add OccupationStandard#inner_hits accessor to store processed results from ES
* Add OccupationStandard#duplicates method to populate similar_results partial 
* Updates partial to use #duplicates instead of #similar_results

image

### CODE REVIEW

**Review by littleforest — COMMENTED** [review]
Looks great! Can you also add an after party task to reindex the OccupationStandard ES model since this makes changes to the index?

File: app/models/occupation_standard.rb [hunk 1]
```diff
@@ -126,6 +128,7 @@ class OccupationStandard < ApplicationRecord
       indexes :work_process_titles, type: :text, analyzer: :english
       indexes :related_job_titles, type: :text, analyzer: :rebuilt_english
       indexes :created_at, type: :date
+      indexes :headline, type: :keyword
```
[the remark this answers is not in the document]
Comment (FerPerales) [reply app/models/occupation_standard.rb:131]: @littleforest agree! We can tackle this on a different PR

[the remark this answers is not in the document]
Comment (littleforest) [reply app/models/occupation_standard.rb:131]: 👍 I'll make a ticket

File: app/controllers/occupation_standards_controller.rb [hunk 2]
```diff
@@ -78,4 +78,15 @@ def current_page
   def offset
     (current_page.to_i - 1) * Pagy::DEFAULT[:items]
   end
+
+  def add_inner_hits_from_results(occupation_standards)
+    occupation_standards.map_with_hit do |occupation_standard, result|
```
Comment (littleforest) [inline app/controllers/occupation_standards_controller.rb:83]: cool that you found this method!

**Review by littleforest — APPROVED** [review]
🚢

**Pull request merged**